### ISSUE TITLE
[FEATURE] - Exporting/importing Yoopta content in different formats (html, md, text)

### ISSUE THREAD

**Issue by Darginec05** [Darginec05/Yoopta-Editor#150]
Package `@yoopta/exports` should be finished with new features in v4.
Supported formats for starts: `html, md, plaint text`

### Discussed in https://github.com/Darginec05/Yoopta-Editor/discussions/137

<div type='discussions-op-text'>

<sup>Originally posted by **nachog00** May  5, 2024</sup>
> Thanks for the great plugin! I'm surely be using it a lot.
> Now my question is:
> is there any way to export the content as a plain html? I intend on using this as a high quality input in a form, and I need to store the input as HTML.
> I saw the editor.getEditorValue method returns an object with a tree but i cant really use that as it is.
> Thanks anyways.</div>

**Comment by michaelshimeles** [comment]
This would be so helpful. This is the only feature stopping me from switching over

**Comment by maZahaca** [comment]
"stopping me from switching over" is the best motivation to work overnight! Let's do it!

**Comment by Darginec05** [comment]
@maZahaca  Okay, challenge accepted!

**Comment by michaelshimeles** [comment]
I'll swap asap the second it's live :)

**Comment by chrassendk** [comment]
@Darginec05 Is there an tentative ETA for a 4.x upgrade of `exports? In order for rolling it out in prod. we need to import export HTML

**Comment by Darginec05** [comment]
Hi @chrassendk 👋  
I'm currently working on it, but parts with html and markdown are already done. 
I have plans to pulbish today @yoopta/exports as `-rc` version

**Comment by maZahaca** [comment]
It seems the most wanted feature now! @Darginec05

**Comment by Darginec05** [comment]
@vlad-svr Roger that.
I ping you when I'll publish `-rc` version

**Comment by Darginec05** [comment]
@chrassendk @michaelshimeles @vlad-svr @maZahaca Good news, guys!

I finally released the -rc version for @yoopta/exports 🔥
I also worked on examples to make it as convenient as possible for developers to understand how deserializers/serializers work. Check examples below 👇

So, what we have in current time: 
 - Version for packages - 4.5.2-rc.1
 - Full docs  - https://yoodocs.space/yoopta-editor/yoopta/exportsimports-fUZcaVYc7w
 - Examples - https://yoopta-editor-jt4typuh8-dargos-projects.vercel.app/examples/withExports
    - Example with HTML - https://yoopta-editor-jt4typuh8-dargos-projects.vercel.app/examples/withExports/html
    - Example with Markdown - https://yoopta-editor-jt4typuh8-dargos-projects.vercel.app/examples/withExports/markdown

But this task is still in progress.
What missing in current version:  - Cut content from editor
 - Copy/cut content from editor in readOnly mode
 - Parsing nested inline marks from markdown

I think these minor bugs will be fixed soon and I'll publish a stable version ASAP.
